app/models/agents/rss_agent.rb

```diff
@@ -113,7 +113,7 @@ def check_urls(urls)
 
       created_event_count = 0
       sort_events(new_events).each.with_index do |event, index|
-        entry_id = event.payload[:id]
+        entry_id = event.payload[:update_id]
```

Comment:
Will the re-fetch cause duplicate events to be emitted for users of RssAgents?  If so, we should probably figure out how to migrate existing users.